src/main/java/com/minecolonies/coremod/sounds/ModSoundEvents.java

```diff
@@ -5,12 +5,20 @@
 import net.minecraft.util.SoundEvent;
 import net.minecraftforge.fml.common.registry.GameRegistry;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Registering of sound events for our colony.
  */
 public final class ModSoundEvents
 {
     /**
+     * List of sound handlers.
+     */
+    public static final List<AbstractWorkerSounds> handlers = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2386)